tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1754,18 +1754,22 @@ def test_metrics_materialization_upgrade_succeeds_and_produces_expected_latest_m
         This JSON dump can be replicated by installing MLflow version 1.2.0 and executing the
         following code from the directory containing this test suite:
 
-        >>> import json
-        >>> import mlflow
-        >>> from mlflow.tracking.client import MlflowClient
-        >>> mlflow.set_tracking_uri(
-        ...     "sqlite:///../../resources/db/db_version_7ac759974ad8_with_metrics.sql")
-        >>> client = MlflowClient()
-        >>> summary_metrics = {
-        ...     run.info.run_id: run.data.metrics for run
-        ...     in client.search_runs(experiment_ids="0")
-        ... }
-        >>> with open("dump.json", "w") as dump_file:
-        >>>     json.dump(summary_metrics, dump_file, indent=4)
+        .. code-block:: python
```

Comment:
This is simply to prevent IDE auto-detection of Docs executable elements when running tests manually in an IDE.